index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+   <meta charset="UTF-8">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.2/css/bootstrap.min.css">
+    <title>User Can View Grades</title>
+    <style>
+      header {
+        font-size: 24px;
+        font-weight: bold;
+      }
+    </style>
```

Comment:
You can accomplish something similar by using a heading (`<h*>`) tag inside the `<header>`, without any custom CSS.